app/client.js

```diff
@@ -101,6 +105,32 @@ async function init() {
 
   const context = await app.rehydrate(window.state);
 
+  if (config.mainMenu.countrySelection) {
+    const selectedCountries = context.getStore('CountryStore').getCountries();
+    const keys = Object.keys(selectedCountries);
+    let boundaries = 'FIN';
+    let feedIds = [...config.feedIds];
+    keys.forEach(key => {
+      if (selectedCountries[key]) {
+        boundaries += `,${getGeocodingCountryCode(key)}`;
+        const countryFeedIds = getCountryFeedId(key);
+        feedIds = [...feedIds, ...countryFeedIds];
+      }
+    });
+
+    if (boundaries !== 'FIN') {
```

Comment:
Default country FIN must be in config, too. UI is used in other countries ass well.